optimize/functions/vlse.go

```diff
@@ -47,6 +47,52 @@ func (Bukin6) Func(x []float64) float64 {
 	return 100*math.Sqrt(math.Abs(x[1]-0.01*x[0]*x[0])) + 0.01*math.Abs(x[0]+10)
 }
 
+// CamelThree implements the three-hump camel function, a two-dimensional function
+// with three local minima and a single global minimum.
+// The function is given by
+//  f(x) = 2*x_0^2 - 1.05*x_0^4 + x_0^6/6 + x_0*x_1 + x_1^2
+// with the global minima at
+//  x^* = (0, 0)
+//  f(x^*) = 0
+// The typical domain is x_i ∈ [-5, 5] for all i.
+// Reference:
+//  https://www.sfu.ca/~ssurjano/camel3.html (obtained December 2017)
+type CamelThree struct{}
+
+func (c CamelThree) Func(x []float64) float64 {
+	if len(x) != 2 {
+		panic("camelthree: dimension must be 2")
+	}
+	x0 := x[0]
+	x1 := x[1]
+	x02 := x0 * x0
+	x04 := x02 * x02
+	return 2*x02 - 1.05*x04 + x04*x02/6 + x0*x1 + x1*x1
+}
+
+// CamelSix implements the six-hump camel function, a two-dimensional function
+// with six local minima and two global minima.
+// The function is given by
+//  f(x) = (4 - 2.1*x_0^2 + x_0^4/3)*x_0^2 + x_0*x_1 + (-4 + 4*x_1^2)*x_1^2
+// with the global minima at
+//  x^* = (±0.0898, -0.7126)
```

Comment:
Is there a Unicode symbol for "minus over plus"? It should be in front of 0.7126 instead of the minus.